### DISCUSSION TITLE
What are the different release executables for? icloud, icloudpd, & icloudpd-ex

### DISCUSSION THREAD

**Discussion by turpie** [icloud-photos-downloader/icloud_photos_downloader#703] [Q&A]
What are the different release executables for? 
The are versions starting with "icloud", "icloudpd", & "icloudpd-ex". Is there any reason to choose one over the other?

**Comment by AndreyNikiforov** [comment]
`icloudpd` is the one you need to download photos. `icloud` is a tool to maintain passwords in keyring. `icloudpd-ex` is an [Experimental](https://github.com/icloud-photos-downloader/icloud_photos_downloader#experimental-mode) version of the `icloudpd`
